test/unit/feature-test.js

```diff
@@ -1,14 +1,16 @@
-describe('A feature test', function() {
-  beforeEach(function() {
+import MyLibrary from '../../src/index';
+
+describe('A feature test', () => {
+  beforeEach(() => {
     spy(MyLibrary, 'mainFn');
     MyLibrary.mainFn();
   });
 
-  it('should have been run once', function() {
+  it('should have been run once', () => {
     expect(MyLibrary.mainFn).to.have.been.calledOnce;
   });
 
-  it('should have always returned hello', function() {
+  it('should have always returned hello', () => {
```

Comment:
Yeah? I've been writing a whole bunch of (aka: all) unit tests in this syntax for awhile now, and I haven't had any problemos with Mocha. What sorts of problems are you running into?